ddmore.Rmd

```diff
@@ -20,10 +20,23 @@ data_dir <- "./testdata/nmoutputs"
 file_paths <- list.files(data_dir, "\\.lst$", full.names = TRUE)
 bdf <- function(.filename){
   df <- data.frame( stringsAsFactors = FALSE,
-                    filename = as.character(basename(.filename)),  
```

Comment:
can initialize with `tibble::tibble()` instead of data.frame, and you also then don't need stringsAsFactors = FALSE as that is the default for tibbles

Reply:
ok